sdcm/tester.py

```diff
@@ -237,8 +237,17 @@ def get_nemesis_class(self):
         :return: Nemesis class.
         :rtype: nemesis.Nemesis derived class
         """
-        class_name = self.params.get('nemesis_class_name')
-        return getattr(nemesis, class_name)
+        nemesis_threads = []
+        list_class_name = self.params.get('nemesis_class_name')
+        for cl in list_class_name.split(' '):
+            try:
+                nemesis_name, num = cl.split(':')
```

Comment:
I would advice to use .strip() on outputs, so extra spaces (or any other unprintable chars like \n or \t) won't break this parsing code.

Reply:
done